test/integration/borrower_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class BorrowerTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  include FactoryGirl::Syntax::Methods
+
+  test "a borrower can view a home page" do
+    visit root_path
+
+    assert page.has_content?("Keevahh")
+  end
+
+  test "a borrower can see all their projects" do
+    borrower = create(:user)
+    tenant = create(:tenant)
+    tenant.items << create(:item)
+    borrower.tenant_id = tenant.id
+    ApplicationController.any_instance.stubs(:current_user).returns(borrower)
+
+    visit tenant_dashboard_path(slug: tenant.slug)
+
+    tenant.items.each do |item|
+      assert page.has_content?(item.title)
+    end
+  end
+
+  test "a borrower is redirected to their dashboard upon login" do
+  end
```

Comment:
I wanted to put in placeholders, but get this merged in, so that anyone could pick it up and continue.